src/components/TrackFilePicker.js

```diff
@@ -0,0 +1,76 @@
+import React, {useState} from "react";
+import PropTypes from "prop-types";
+import Select from "react-select";
+
+
+
+//handleInputChange takes in a string trackType
+export const TrackFilePicker = ({
+  fileOptions, // array of string representing files names
+  fileSelect, 
+  handleInputChange,
```

Comment:
Should we call this `onChange` to match the naming in the `TrackTypeDropown`?

Also, do we want it to take a whole track object instead of just a filename?